Missing bigmpiconf.h?

While making it, I get the below error:
/BigMPI/src/bigmpi_impl.h:4:24: fatal error: bigmpiconf.h: No such file or directory

Is there a file missing somewhere or am I doing it wrong? I am using the master latest branch on CentOS with Mpich 3.2.

Thanks!

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -1,2 +1,8 @@
+add_custom_command(
+  OUTPUT "${CMAKE_CURRENT_SOURCE_DIR}/bigmpiconf.h"
+  COMMAND "./autogen.sh"
+  COMMAND "./configure"
+  WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/..")
+
 file(GLOB SOURCES "${CMAKE_CURRENT_SOURCE_DIR}/*.c")
-add_library(bigmpi ${LIB_LINKAGE_TYPE} ${SOURCES})
+add_library(bigmpi ${LIB_LINKAGE_TYPE} ${SOURCES} "bigmpiconf.h")

--- a/test/CMakeLists.txt
+++ b/test/CMakeLists.txt
@@ -1,5 +1,7 @@
 file(GLOB TESTS "test_*.c")
 
+include_directories("${CMAKE_CURRENT_SOURCE_DIR}/../src")
+
 foreach(test ${TESTS})
   string(REPLACE ".c" "" buffer0 ${test})
   string(REPLACE "${CMAKE_SOURCE_DIR}" "" buffer1 ${buffer0})

